e2e/Makefile

```diff
@@ -0,0 +1,24 @@
+runTestCase:
+	@T="$$(KUBECONFIG="$$ROOT_DIR/kubeconfig" kuttl test --timeout 300 --skip-delete --namespace "$$NAMESPACE" "$$TC" 2>&1)" ;\
+	echo "$$T" |\
+		grep -v harness.go |\
+		grep -v ^=== |\
+		grep -v "running without a 'kuttl-test.yaml" |\
+		grep -v "creation of user-supplied namespace" |\
+		grep -v "not match file name regexp" |\
+		grep -v "kutt-test config testdirs is overridden" ;\
+	echo "$$T" | \
+		grep '^PASS' || \
+			(echo "$$T" | grep harness.go && cat "$$TC/step/00-step.yaml" && exit 1)
+
+renderTestCase:
+	@D="$$(mktemp -d)" ;\
+	mkdir -p "$$D/step" ;\
+	envsubst -i "$$TPL" -o "$$D/step/00-step.yaml" ;\
+	echo -n "$$D"
```

Comment:
When I print the value of `TC`, it prints as `-n /var/folders/c3/y7njgw_x373gsj4nydxmjgz40000gq/T/tmp.af02uAbM` (treating `-n` as a literal). `echo "$$D\c" worked fine here as a replacement.

Reply:
@mhmxs - I didn't see that you merged already. I'll put up a separate PR.